src/Mod/Arch/archviewproviders/view_reinforcement_individual.py

```diff
@@ -0,0 +1,56 @@
+# ***************************************************************************
+# *   Copyright (c) 2020 Bernd Hahnebach <bernd@bimstatik.org>              *
+# *                                                                         *
+# *   This program is free software; you can redistribute it and/or modify  *
+# *   it under the terms of the GNU Lesser General Public License (LGPL)    *
+# *   as published by the Free Software Foundation; either version 2 of     *
+# *   the License, or (at your option) any later version.                   *
+# *   for detail see the LICENCE text file.                                 *
+# *                                                                         *
+# *   This program is distributed in the hope that it will be useful,       *
+# *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
+# *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
+# *   GNU Library General Public License for more details.                  *
+# *                                                                         *
+# *   You should have received a copy of the GNU Library General Public     *
+# *   License along with this program; if not, write to the Free Software   *
+# *   Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  *
+# *   USA                                                                   *
+# *                                                                         *
+# ***************************************************************************
+
+__title__ = "FreeCAD individual reinforcement view object"
+__author__ = "Bernd Hahnebach"
+__url__ = "http://www.freecadweb.org"
+
+import ArchComponent
+
+from .view_reinforcement_generic import ViewProviderReinforcementGeneric
+
+
+class ViewProviderReinforcementIndividual(ViewProviderReinforcementGeneric):
+
+    def getIcon(
+        self
+    ):
+        import Arch_rc
+        False if Arch_rc.__name__ else True  # dummy usage
+        return ":/icons/Arch_ReinforcementIndividual.svg"
+
+    def claimChildren(
+        self
+    ):
+        # collect the children for TreeView
+
+        # children from Arch.Component
+        # since we overwrite the method we need to explicit call it
+        children = ArchComponent.ViewProviderComponent.claimChildren(self)
+
+        # reinforcement individual children
+        if hasattr(self, "Object"):
+            return children + self.Object.Vertieces
```

Comment:
Error? `self.Object.Vertices`

Reply:
No because of https://github.com/berndhahnebach/FreeCAD_bhb/blob/04bf1d52a2a5373876adea16ff852cfe3e516303/src/Mod/Arch/archobjects/reinforcement_individual.py#L54

but I may choose a better property name anyway

Reply:
If you mean the plural of `Vertex`, it is `Vertices`, not `Vertieces`.

Reply:
Yeah not a good property name and a type on top ...

renamed the property to Individuals